app/views/admin/registrations/index.html.haml

```diff
@@ -3,14 +3,34 @@
     .page-header
       %h1
         Registrations
-        = "(#{@registrations.length})" if @registrations
+        = "(#{@total_reg})" if @registrations
```

Comment:
Would be awesome if that variable name wasn't abbreviated, so that I don't have to think twice when I am reading it.
`@total_reg`  also just holds `@registrations.count`, so do we really need to define a new variable for it?

Either way, since we have `@registrations` already loaded, we can use `.length` instead of `.count`, which triggers a new count query on our DB.